spec/controllers/user_sessions_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe UserSessionsController do
+  let(:user) { create(:user, activated_at: Time.new(2007, 1, 1)) }
+
+  context "create" do
+    context "after failed to login more than consecutive login limit" do
+      before do
+        allow(UserSession).to receive(:consecutive_failed_logins_limit).and_return(2)
+        3.times do
```

Comment:
Since you mocked `consecutive_failed_logins_limit` as 2, `2.times` here is enough for the next post to fail. Can you make "2" a local variable?